db/seeds/run-seed.js

```diff
@@ -1,4 +1,5 @@
 const devData = require('../data/development-data/index.js');
+const testData = require('../data/test-data/index.js')
```

Comment:
Do you ever use this? Seems like you can delete it

Reply:
I've been using it to manual test in insomnia, I'll likely swap back and forth on every ticket. Plan is to remove at the end rather than keep adding and readding